github/issues_assignees_test.go

```diff
@@ -96,3 +97,63 @@ func TestIssuesService_IsAssignee_invalidOwner(t *testing.T) {
 	_, _, err := client.Issues.IsAssignee("%", "r", "u")
 	testURLParseError(t, err)
 }
+
+func TestIssuesService_AddAssignees(t *testing.T) {
+	setup()
+	defer teardown()
+
+	mux.HandleFunc("/repos/o/r/issues/1/assignees", func(w http.ResponseWriter, r *http.Request) {
+		var assignees struct {
+			Assignees []string `json:"assignees,omitempty"`
+		}
+		json.NewDecoder(r.Body).Decode(&assignees)
+
+		testMethod(t, r, "POST")
+		testHeader(t, r, "Accept", mediaTypeMultipleAssigneesPreview)
+		want := []string{"user1", "user2"}
+		if assignees.Assignees == nil || !reflect.DeepEqual(assignees.Assignees, want) {
```

Comment:
Is `assignees.Assignees == nil` check necessary? Wouldn't `!reflect.DeepEqual(assignees.Assignees, want)` catch that anyway?